supabase/seed.sql

```diff
@@ -177,19 +186,20 @@ INSERT INTO "public"."conversations" ("id", "created_at", "member_has_deleted")
 
 
 --
 -- Data for Name: items; Type: TABLE DATA; Schema: public; Owner: postgres
 --
 
 INSERT INTO "public"."items" ("id", "created_at", "donated_by", "reserved", "size", "brand", "postcode", "condition", "item_type", "item_subtype", "reserved_by", "item_name", "item_description", "imageSrc", "postable", "collectible", "requestedToReserve", "postage_covered") VALUES
-	(1, '2024-06-16 13:34:25.931798+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, '9', NULL, 'CF1', 'Good', 'shoes', 'men', NULL, 'test_mens_shoes_willing_to_post_name', 'test_mens_shoes_willing_to_post_description', 'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85/ebb2e026-3a94-48db-9405-2fff9530fa84', true, false, NULL, false),
-	(2, '2024-06-16 13:37:00.460318+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, 'S', NULL, 'CF1', 'Fair', 'clothing', 'women', NULL, 'test_womens_clothing_pick_up_name', 'test_womens_clothing_pick_up_description', 'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85/e4611e6a-2682-4c13-9a78-f4132904356d', false, true, NULL, false),
-	(3, '2024-06-16 13:38:30.028263+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, '', NULL, 'CF1', 'Poor', 'toys', '', NULL, 'test_toys_postage_covered_name', 'test_toys_postage_covered_description', 'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85/1f216cae-2721-48ee-9b37-84a8eac94ba0', false, false, NULL, true),
-	(5, '2024-06-16 13:41:48.120062+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, '', NULL, 'CF1', 'New', 'household', '', NULL, 'test_home_pick_up_name', 'test_home_pick_up_description', 'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85/805480d3-d26d-4dad-bc42-c179e9eb8b2a', false, true, NULL, false);
+	(1, '2024-06-16 13:34:25.931798+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, '9', NULL, 'CF1', 'Good', 'shoes', 'men', NULL, 'test_mens_shoes_willing_to_post_name', 'test_mens_shoes_willing_to_post_description', '/public/shoes.jpg', true, false, NULL, false),
+	(2, '2024-06-16 13:37:00.460318+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, 'S', NULL, 'CF1', 'Fair', 'clothing', 'women', NULL, 'test_womens_clothing_pick_up_name', 'test_womens_clothing_pick_up_description', '/public/blouse.jpg', false, true, NULL, false),
+	(3, '2024-06-16 13:38:30.028263+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, '', NULL, 'CF1', 'Poor', 'toys', '', NULL, 'test_toys_postage_covered_name', 'test_toys_postage_covered_description', '/public/toy.jpg', false, false, NULL, true),
+	(5, '2024-06-16 13:41:48.120062+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, '', NULL, 'CF1', 'New', 'household', '', NULL, 'test_home_pick_up_name', 'test_home_pick_up_description', '/public/sofa.jpg', false, true, NULL, false);
```

Comment:
```suggestion
(1, '2024-06-16 13:34:25.931798+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, '9', NULL, 'CF1', 'Good', 'shoes', 'men', NULL, 'Little worn shoes', 'Only wore them twice or three times. Very good state.', 'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/test_images/shoes.jpg?t=2024-07-08T12%3A47%3A53.596Z', true, false, NULL, false),
	(2, '2024-06-16 13:37:00.460318+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, 'S', NULL, 'CF1', 'Fair', 'clothing', 'women', NULL, 'Summer Blouse', 'Clearing out any pieces I that are not needed in my wardrobe any more. This blouse is in pretty good state except for a missing button.', 'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/test_images/blouse.jpg?t=2024-07-08T12%3A48%3A10.214Z', false, true, NULL, false),
	(3, '2024-06-16 13:38:30.028263+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, '', NULL, 'CF1', 'Poor', 'toys', '', NULL, 'Teddy bear', 'This teddy brought my baby a lot of happiness and I hope it will do the same for someone else in a new home :)', 'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/test_images/toy.jpg?t=2024-07-08T12%3A48%3A39.092Z', false, false, NULL, true),
	(5, '2024-06-16 13:41:48.120062+00', 'cab97c8e-6704-4bb2-b3e8-a9cdaaf1be85', false, '', NULL, 'CF1', 'New', 'household', '', NULL, 'Brand new sofa', 'Unfortunately I have no way of transporting this out to anyone so you will need to be able to pick it up, but the sofa is an brand new condition.', 'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/test_images/sofa.jpg?t=2024-07-08T12%3A48%3A35.712Z', false, true, NULL, false);
```

I played around with it on my local and by changing these lines I can see the item images fine now that we've hosted them in staging